bc/channel/models.py

```diff
@@ -130,12 +127,10 @@ def self_url(self):
             case self.TWITTER:
                 return f"https://twitter.com/{self.account}"
             case self.MASTODON:
-                result = masto_regex.search(self.account)
-                assert len(result.groups()) == 2
-                account_part, instance_part = result.groups()
-                return f"https://{instance_part}/@{account_part}"
+                account_part, instance_part = get_handle_parts(self.account)
+                return f"{instance_part}@{account_part}"
             case self.BLUESKY:
-                return f"https://bsky.app/profile/{self.account_id}"
+                return f"https://bsky.app/profile/{self.account}"
```

Comment:
Let's use the `account_id` because we store handles without the "@" symbol in there.

Reply:
@ERosendo done!